### PR TITLE
Content: Stubbing SAP FICO track

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

This track sets up a generic set of blocks called entry-tracks, which genericises much of the piscine organisational and operational content.

This is then worked through using the curriculum breakdown plan for a potential upcoming SAP/FICO track to input all the specifics. This track is in draft, so I have only provided the learning objectives and setup content from this doc.

### This means I have 
- created a repo https://github.com/CodeYourFuture/Track-ERP-SAP-FICO
- created workshops and projects folders in that repo
- stubbed workshops and projects with readmes
- linked those readmes in the day plan
- created an issue for each sprint https://github.com/CodeYourFuture/Track-ERP-SAP-FICO/issues
- assigned that repo in the backlog view
- added the learning objectives, entry, and exit criteria

https://docs.google.com/document/d/15gMYMv2Rs9sMbdUpINul7kww6ejZDiEXOe38ShuU3j8/

### Common Content?

Yes  - a whole mess of generic blocks in modules/entry-tracks

### Org Content?

Tracks | SAP-FICO | all

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: org-cyf-tracks/content/blocks/sap-fico/prereqs/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+]
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
++++
+
+_To be selected for this course a learner should be able to..._
+
+- [ ] Complete Step 3 of CYF Introduction to Programming or JS2 of the pre-2025 SDC
```
Comment (illicitonion) [inline org-cyf-tracks/content/blocks/sap-fico/prereqs/index.md:17]: Not Step 4? Or "the whole course"?

File: common-content/en/module/entry-tracks/help/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  question="Who will help you with your blockers?"
+  answers="Nobody, I must do everything myself | Only volunteers can help me | We all help each other in our community"
+  feedback="No, we work in teams. | No, your team can help you too. | Yes, a quality professional builds understanding by asking and answering good questions."
+  correct="2" >}}
+
+{{<note type="tip" title="Remember">}}
+
+We all help each other at {{<our-name>}}. We are a community.
+
+You are not competing against each other for limited number of places. We will accept everyone who meets the success criteria.
```
Comment (illicitonion) [inline common-content/en/module/entry-tracks/help/index.md:44]:
```suggestion
You are not competing against each other for a limited number of places on the track. We will accept everyone who meets the success criteria.
```

File: common-content/en/module/entry-tracks/goals/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  render = 'never'
+  list = 'local'
+  publishResources = false
++++
+
+Your goal is to complete at least one team project, four team workshops, and the assigned coursework. For each activity, your goal is to:
+
+- 🎯 Build a complete understanding of the work you have done, because you will be asked to explain it.
+
+_Our_ goal is to find people who are ready to join a service desk and hit the ground running. This means we are looking for people ready to begin working as an IT professional within the next 8 weeks. This is why we are looking at your ability to:
```
Comment (illicitonion) [inline common-content/en/module/entry-tracks/goals/index.md:19]: This says service desk, but I don't think this is a service desk track?

**Review by illicitonion — APPROVED** [review]
LGTM! (Also the objectives/tasks aren't rendering right, but there's already an issue for that)

File: org-cyf-tracks/content/blocks/sap-fico/prereqs/index.md (hunk 1 above)
Comment (SallyMcGrath) [reply org-cyf-tracks/content/blocks/sap-fico/prereqs/index.md:17]: Talk to @gbencci about that one.

**Pull request merged**